src/attr.rs

```diff
@@ -301,13 +315,16 @@ fn error_expected_args(attr: &Attribute) -> Error {
     let msg = format!("expected attribute arguments: {}[{}(...)]", style, path);
 
     #[cfg(feature = "printing")]
+    #[cfg_attr(syn_enable_doc_cfg, doc(cfg(feature = "printing")))]
```

Comment:
We don't need doc attributes on `cfg`-ed statements either, as they're not documented or exported.